### ISSUE TITLE
Reduce delay for 'More Details' (also change case of the D)

### ISSUE THREAD

**Issue by adambard1** [streetlives/yourpeer.nyc#169]
- [ ] On Mobile or desktop, the delay after clicking More Details to get to the Location details page was significant enough to make me think it was broken and tap again a couple of times. I was connected to my wifi but it was the same for 5G.

https://github.com/user-attachments/assets/4a0b7361-f417-4296-bac5-d97ca2293c01

**Comment by jbeard4** [comment]
This is acknowledged. Will investigate.

**Comment by jbeard4** [comment]
This should be improved by https://github.com/streetlives/yourpeer.nyc-nextjs/pull/19

We are at about 1.5s page load time. I'm going to continue optimizing it until it's < 1s, but this will come later.

**Comment by adambard1** [comment]
It's improved. I understand further improvement is possible. Seems ready to deploy. Appreciating it's a server thing but lifting up that I haven't tested the deployment on 4G, which may provide other delays if e.g., the image file for the homepage hero is still large.

**Comment by jbeard4** [comment]
I don't think that 4G is going to to make a difference, as the delay is not caused by large payload, but by a slow query on the server. But we can revisit this and optimize the query.
